app/models/post.rb

```diff
@@ -14,4 +14,8 @@ def recent_comments
   def update_posts_counter
     author.update(post_counter: author.posts.count)
   end
+
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 end
```

Comment:
- [ ] Kindly also do the same and add tests for the `recent_comments` and `update_posts_counter` methods in your `Post` model.
- [ ]  You might also want to check your `Comments` model and add unit tests for the `update_comments_counter`  custom method

Reply:
@frank1738 Thanks for the review. I just wanted to clear out a confusion I have, the `update_posts_counter` and `update_comments_counter` are defined privately, do I still need to test the private instance methods?

Reply:
> @frank1738 Thanks for the review. I just wanted to clear out a confusion I have, the `update_posts_counter` and `update_comments_counter` are defined privately, do I still need to test the private instance methods?

@rubydevi , Since the project requires tests for all model methods and validations, we'll cover both public and private methods. Let's make sure we test every method, including private methods. 🚀🧪

Reply:
Thanks for clearing that one up for me :D